code/modules/mob/living/simple_animal/friendly/mule_bot/mule_bot.dm

```diff
@@ -0,0 +1,202 @@
+
+/obj/effect/mule_link/Initialize()
+	. = ..()
+	var/mob/living/simple_animal/mule_bot/M = new(loc)
+	var/obj/item/remote/R = new(loc)
+	M.try_link(R)
+	R.bot = M
+	qdel(src)
+
+
+/mob/living/simple_animal/mule_bot
+	name = "Felidae Beetle MK 1"
+	desc = "A highly spohisticated load carrying robotic companion for the advanced marine. Manly as hell!"
+	icon = 'icons/mob/kerfus.dmi'
+	icon_state = "kerfus_blank"
+	icon_living = "kerfus_blank"
+	icon_dead = "kerfus"
+	gender = FEMALE
+	speak = list("Meow!", "Purr!", "Remember to stock up on medicine!", "Ill carry for the team!", "I sure do hope all these munations are safe...",
+	"Did we really need to bring THAT many plasma cutters?", "Oh I may have the thing for that!","I don't think a xenomorphs head would fit...","Sometimes I wonder if moving bullets is the best way to make a living..","Please stop calling me a tin can...","Time to restock!")
+	speak_emote = list("purrs", "meows")
+	emote_hear = list("meows.", "mews.")
+	emote_see = list("shakes its head.", "shivers.")
+	speak_chance = 5
+	turns_per_move = 5
+	see_in_dark = 6
+	flags_pass = PASSTABLE | PASSMOB | PASSXENO //to avoid it body blocking marines and xenos
+	a_intent = INTENT_HELP
+	mob_size = MOB_SIZE_SMALL
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	a_intent = INTENT_HELP
+	light_system = MOVABLE_LIGHT
+	flags_atom = BUMP_ATTACKABLE //bump attack may not work due to pass flags
+	light_color =  "#3262db"
+	light_range = 2
+	light_power = 1
+	light_on = FALSE
```

Comment:
Iirc did this not look ugly

Reply:
the light is off, but ill remove the vars since those are not required